src/sidebar/util/copy-to-clipboard.ts

```diff
@@ -30,3 +32,35 @@ export function copyText(text: string) {
     temp.remove();
   }
 }
+
+/**
+ * Copy the string `text` to the clipboard verbatim.
+ *
+ * @throws {Error}
+ *   This function may throw an error if the `clipboard-write` permission was
+ *   not allowed.
+ */
+export async function copyPlainText(text: string, navigator_ = navigator) {
+  await navigator_.clipboard.writeText(text);
+}
+
+/**
+ * Copy the string `text` to the clipboard, rendering HTML if any, instead of
+ * raw markup.
+ *
+ * If the browser does not support this, it will fall back to copy the string
+ * as plain text.
```

Comment:
I think this is a bit of a footgun, as the fallback behavior here is not useful to non-technical end users. I think it would be better to fail if the media type cannot be set to HTML. At a higher level, there should be a way for the caller to detect support, in a way that is consistent with `copyHTML`, and disable the option to copy.

Reply:
I'll handle this as part of https://github.com/orgs/hypothesis/projects/131/views/1?pane=issue&itemId=50124775, which basically means that if statement will fall back to `document.execCommand('copy')`, instead of copying plain text.